plugins/action/dcnm_inventory.py

```diff
@@ -41,17 +41,17 @@ def run(self, tmp=None, task_vars=None):
 
         timeout = 1000
 
-        display.v(
-            "PERSISTENT_COMMAND_TIMEOUT is %s"
-            % str(persistent_command_timeout),
-            self._play_context.remote_addr,
-        )
-        display.v(
-            "PERSISTENT_CONNECT_TIMEOUT is %s"
-            % str(persistent_connect_timeout),
-            self._play_context.remote_addr,
-        )
         if (persistent_command_timeout < timeout or persistent_connect_timeout < timeout):
+            display.warning(
```

Comment:
@kharicha FYI, I added this update to your diffs since it's in the `dcnm_inventory` action plugin.  This does three things.

* Changes the display messages to warnings so they show up even if `-v` is not specified when running the playbook.
* Only displays the warning message about adding switches if we are actually adding switches.
* Moves the warning about timeout settings so they are only displayed if the timers are too low.